Extension/package.nls.json

```diff
@@ -77,6 +77,7 @@
     "c_cpp.debuggers.pipeTransport.pipeProgram.description": "The fully qualified pipe command to execute.",
     "c_cpp.debuggers.pipeTransport.pipeArgs.description": "Command line arguments passed to the pipe program to configure the connection.",
     "c_cpp.debuggers.pipeTransport.pipeEnv.description": "Environment variables passed to the pipe program.",
+    "c_cpp.debuggers.pipeTransport.quoteArgs.description": "If individual arguments contain characters (such as spaces or tabs), should it be quoted? If 'false', the debugger command will no longer be automatically quoted. \nDefault is 'true'.",
```

Comment:
I think we talked about rewording it to `If the pipeProgram's individual arguments...`